### DISCUSSION TITLE
import { create } from 'zustand' raises Error in typescript

### DISCUSSION THREAD

**Discussion by geyang** [pmndrs/zustand#2688] [Bug report]
### Bug Description

In README, it says

```typescript
import { create } from 'zustand'

const useBearStore = create((set) => ({
  bears: 0,
  increasePopulation: () => set((state) => ({ bears: state.bears + 1 })),
  removeAllBears: () => set({ bears: 0 }),
}))
```

However, typescript raises a warning. It sugguests we use 
```typescript
import create from 'zuztand'
```

instead. 
<img width="471" alt="Screenshot 2024-08-18 at 4 20 21 PM" src="https://github.com/user-attachments/assets/75fbb51c-ec41-4bab-90a1-0aa9fcd2f8c8">
<img width="521" alt="Screenshot 2024-08-18 at 4 20 19 PM" src="https://github.com/user-attachments/assets/b0b79a82-982d-431d-99c7-be87364b693f">

### Reproduction Link

https://stackblitz.com/edit/react-ts-qzvl2u?file=index.tsx

**Comment by dbritto-dev** [comment]
@geyang you are using an old version of zustand

**Reply by geyang** [reply]
I just upgraded. There is till a warning when using destructured import. Has others run into similar issues?

**Reply by dbritto-dev** [reply]
@geyang I don't get any issue on latest version
